app/views/admin/users/notes/_index.html.erb

```diff
@@ -8,13 +8,15 @@
   <% else %>
     <div class="pt-3">
       <% @notes.each do |note| %>
+        <% author = User.find(note.author_id).username %>
```

Comment:
We should having this sort of logic in the views, but I know its out of scope of this PR so I'll add a note to fix it in a subsequent PR :)